renderer/utils/reports.ts

```diff
@@ -188,3 +191,28 @@ export function formatDuration(ms: number): string {
   }
   return `${Math.floor(hours * 100) / 100}h`;
 }
+
+export function checkIntersection(previousTo: string, currentFrom: string) {
+  const [hoursTo, minutesTo] = previousTo.split(":");
+  const [hoursFrom, minutesFrom] = currentFrom.split(":");
+  const toInMinutes = Number(hoursTo) * 60 + Number(minutesTo);
+  const fromInMinutes = Number(hoursFrom) * 60 + Number(minutesFrom);
+  return fromInMinutes < toInMinutes;
+}
+export function validation(activities: Array<ReportActivity>) {
+  for (let i = 0; i < activities.length; i++) {
+    if (i > 0 && checkIntersection(activities[i - 1].to, activities[i].from)) {
+      activities[i - 1].isValid = false;
+    }
+    if (
+      activities[i].duration <= 0 ||
+      (activities[i].project && !activities[i].to)
+    ) {
+      activities[i].isValid = false;
+    }
+    if (activities[i].description.startsWith("!")) {
+      activities[i].mistakes += " startsWith!";
+    }
+  }
```

Comment:
can we add one more check like: 
    if (i > 0 && !activities[i].to && !activities[i].project) {
      activities[i - 1].isValid = false;
    }
to prvent reports like:
...
09:00 - 
10:00 -